audb/core/dependencies.py

```diff
@@ -308,24 +331,22 @@ def load(self, path: str):
             )
         if extension == "pkl":
             self._df = pd.read_pickle(path)
+
         elif extension == "csv":
-            # Data type of dependency columns
-            dtype_mapping = {
-                name: dtype
-                for name, dtype in zip(
-                    define.DEPEND_FIELD_NAMES.values(),
-                    define.DEPEND_FIELD_DTYPES.values(),
-                )
-            }
-            # Data type of index
-            index = 0
-            self._df = pd.read_csv(
+            table = csv.read_csv(
                 path,
-                index_col=index,
-                na_filter=False,
-                dtype=dtype_mapping,
+                read_options=csv.ReadOptions(
+                    column_names=self._schema.names,
+                    skip_rows=1,
+                ),
+                convert_options=csv.ConvertOptions(column_types=self._schema),
             )
-            self._df.index.name = None
+            self._df = self._table_to_dataframe(table)
+
+        elif extension == "parquet":
+            table = parquet.read_table(path)
+            self._df = self._table_to_dataframe(table)
```

Comment:
Thanks for pointing `polars` out. I did indeed a few tests with it, but it was slightly slower to `pyarrow`, and seems also to use `pyarrow.Table` internally or at least converts to it before converting to `pandas`, compare the documentation of [polars.DataFrame.to_pandas()](https://docs.pola.rs/py-polars/html/reference/dataframe/api/polars.DataFrame.to_pandas.html).
Please open an issue, if you which to add it to the [comparison benchmark](https://github.com/audeering/audb/blob/dev/benchmarks/README.md).

Your first comment I do not understand completely. Should I replace `self._table_to_dataframe()` with something else, or extend it?

Reply:
>> Your first comment I do not understand completely. Should I replace self._table_to_dataframe() with something else, or extend it?

This was only a question and an introduction to the section about the various "flavors" of parquet files that could arise (pyarrow versus polars created, usage of a compression approach). So nothing needs to be done there. 

Gainig experience with polars by adding it as a benchmark makes sense. Afaik it is good in memory efficiency and preferred for streaming purposes, so it might become relevant in the future. Saying that it would not need to be implemented right now but allows to resolve this conversation.